src/array/useSetState.ts

```diff
@@ -15,7 +18,15 @@ export function useSetState<T extends object>(initialValue: T): UseSetState<T> {
     },
     [setValue],
   );
-  return [value, setState];
+  const resetState = useCallback(() => setValue(initialValue), []);
+  const actions = useMemo(
+    () => ({
+      setState,
+      resetState,
+    }),
+    [setState, resetState],
+  )
+  return [value, actions];
```

Comment:
I would suggest you add resetState as a third element into the array to avoid breaking changes.

Reply:
ameded by the latest commit